src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,21 +2,21 @@
 
 public class IntLinkedList {
 
-	Node last;
-	int i;
+	private Node last;

```

Comment:
'last' shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internallt usually do not require encapsulation.